Explorer/Assets/Scripts/Utility/Primitives/PlaneFactory.cs

```diff
@@ -0,0 +1,83 @@
+using UnityEngine;
+
+namespace Utility.Primitives
+{
+    public static class PlaneFactory
```

Comment:
Consider introducing tests for each primitive factory similarly to `CylinderVariantsShould`: it's important and straightforward to validate the pure math logic and buffers reusing